test/lib/configurable.js

```diff
@@ -0,0 +1,171 @@
+'use strict';
```

Comment:
What's up? It's not in production codebase, and even if it was it wouldn't be a problem.